src/pages/ReimbursementAccount/NonUSD/Country/subSteps/Confirmation.tsx

```diff
@@ -101,7 +101,7 @@ function Confirmation({onNext, policyID}: ConfirmationStepProps) {
             </Text>
             <InputWrapper
                 InputComponent={PushRowWithModal}
-                optionsList={shouldAllowChange ? CONST.ALL_EUROPEAN_COUNTRIES : CONST.ALL_COUNTRIES}
+                optionsList={shouldAllowChange ? CONST.ALL_EUROPEAN_UNION_COUNTRIES : CONST.ALL_COUNTRIES}
```

Comment:
Just wanted to double check if this is expected

Reply:
Thanks for confirming. Everything looks good to me, we will only need the design approved here.